### DISCUSSION TITLE
Whether trees can be artificially planted in FATES.

### DISCUSSION THREAD

**Discussion by niuhanlin** [NGEET/fates#967] [Q&A]
Has anyone thought about how to join the plantation?
The role of plantation forests in the context of climate change seems obvious.
Is there a function, like logging, that artificially increases vegetation in an area?

**Comment by JoshuaRady** [comment]
In default FATES you can not plant trees per se, although depending on what you need to do you can approximate it at the start of a run using either a cold start or inventory initialization.

I developed the [Vegetation Management Module for FATES](https://joshuarady.github.io/VegetationManagement/) for my PhD work that allows you to arbitrarily plant trees at any time in a run (and other things_.  I used this to simulate plantations in the Southeastern United States.  The VM Module is currently in my FATES repository but others have used it.  It you think it would be useful to you, please let me know.

**Comment by niuhanlin** [comment]
@JoshuaRady 
Wow! It's a great job.  More and more studies have proved that tree planting is important for ecosystem restoration.
I have recently been studying the contribution of climate change and anthropogenic impacts to ecological restoration.
I have read the VM module you developed.  
I will do some tests in the study area according to the information you provided.
